Source/WebCore/css/StyleColor.cpp

```diff
@@ -132,6 +132,17 @@ bool StyleColor::containsCurrentColor(const CSSPrimitiveValue& value)
     return false;
 }
 
+bool StyleColor::containsColorSchemeDependentColor(const CSSPrimitiveValue& value)
+{
+    if (StyleColor::isSystemColorKeyword(value.valueID()))
```

Comment:
Does all system colors depend on color scheme ?

Reply:
Not all of them do, and even the ones that do can be platform-specific. On Cocoa platforms, the majority of them are color-scheme dependent.

However, I think that using an allowlist or denylist for that set of system colors would be complicated to maintain, for limited benefit. Both an allowlist and denylist would have the problem of knowing the keep the list up-to-date, as new colors are added, or colors change their dependency on color-scheme.

Let me know if you feel strongly about a more specific list.

Reply:
Indeed, having a platform specific list would require some more code (stuff in `/platform`..etc..) I will follow your judgment for the cost-benefit ratio